wagtail/admin/views/reports.py

```diff
@@ -0,0 +1,30 @@
+from django.utils.translation import ugettext_lazy as _
+from django.views.generic.base import TemplateResponseMixin
+from django.views.generic.list import BaseListView
+
+from wagtail.core.models import UserPagePermissionsProxy
+
+
+class ReportView(TemplateResponseMixin, BaseListView):
+    header_icon = ''
+    page_kwarg = 'p'
+    template_name = None
+    title = ''
+    paginate_by = 10
+
+    def get_context_data(self, *args, object_list=None, **kwargs):
+        context = super().get_context_data(*args, object_list=object_list, **kwargs)
+        context['title'] = self.title
+        context['header_icon'] = self.header_icon
+        return context
+
+
+class LockedPagesView(ReportView):
```

Comment:
Probably worth adding a permission check for the 'unlock any page' permission to this view (in addition to the one to be added to the menu item).

Reply:
Users can unlock pages that they have locked themselves without requiring this permission, should we add that to the check too?